components/ActionSheet/src/Theming/MDCActionSheetController+MaterialTheming.m

```diff
@@ -36,8 +37,13 @@ - (void)applyThemeWithScheme:(id<MDCContainerScheming>)scheme {
   [self applyThemeWithTypographyScheme:typographyScheme];
 }
 
+- (UIColor *)dynamicColorWithColor:(UIColor *)color elevation:(CGFloat)elevation {
```

Comment:
This function too general enough to live in an MDCActionSheetController extension

Reply:
Resolved in offline convo – will be addressed in a future PR